clientconn.go

```diff
@@ -954,8 +954,8 @@ func (ac *addrConn) updateConnectivityState(s connectivity.State) {
 func (ac *addrConn) adjustParams(r transport.GoAwayReason) {
 	switch r {
 	case transport.GoAwayTooManyPings:
-		v := 2 * ac.dopts.copts.KeepaliveParams.Time
 		ac.cc.mu.Lock()
+		v := 2 * ac.dopts.copts.KeepaliveParams.Time
```

Comment:
This doesn't seems necessary to me. `ac.mu` is held before `adjustParams` is called. `ac.cc.mu` doesn't affect anything in this line.

The race seems to only exist in the old version.